sdk/mediaservices/arm-mediaservices/README.md

```diff
@@ -24,25 +24,40 @@ npm install @azure/arm-mediaservices
 npm install @azure/ms-rest-nodeauth@"^3.0.0"
 ```
 
-##### Sample code
+##### Sample code using Service Principal to list all Assets
```

Comment:
A few lines above, there is a header that has reference to "list accountFilters". That should get updated to "list assets".